src/saving/Saving Status Fade Out.tres

```diff
@@ -0,0 +1,56 @@
+[gd_resource type="Animation" format=2]
```

Comment:
You don't need to save the animation externally, I think just embedding it inside the SaveStatusOverlay.tscn is more cleaner because the animation is not used elsewhere